test/models/user_test.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+require 'yaml'
```

Comment:
use rspec for all test

Reply:
@kth-wtag Apu, I did. These are my initial tests before introducing `RSpec`. These tests will be removed in future PRs.